appveyor.yml

```diff
@@ -30,6 +32,13 @@ install:
   - C:\projects\julia-binary.exe /S /D=C:\projects\julia
 
 build_script:
-  - C:\projects\julia\bin\julia -e "versioninfo();
-      Pkg.clone(pwd(), \"IJulia\"); Pkg.build(\"IJulia\")"
-#  - C:\projects\julia\bin\julia -e "Pkg.test(\"IJulia\")"
+  - C:\projects\julia\bin\julia -F -e "versioninfo(true);
+      Pkg.clone(pwd(), \"IJulia\");
+      using Conda;
+      try Conda.add(\"qt\") end;
+      postlink = normpath(Pkg.dir(\"Conda\"), \"deps/usr/pkgs/qt-4.8.7-vc9_9/Scripts/.qt-post-link.bat\");
```

Comment:
right. it's just to pass the appveyor. I don't know exactly how to resolve that correctly.
I thought that this pull request has too much things,
I hope to open another commit/issue/pull request for this responsibility.